### PR TITLE
chore(CHANGELOG): update to v1.0.0-alpha.11

### PR DESCRIPTION
**Pull request by philippfromme**
# Changes

## 1.0.0-alpha.11

* `FEAT`: add show callbacks to show erro code and message name errors ([#654](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/654))

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ All notable changes to [bpmn-js-properties-panel](https://github.com/bpmn-io/bpm
 
 ___Note:__ Yet to be released changes appear here._
 
+## 1.0.0-alpha.11
+
+* `FEAT`: add show callbacks to show erro code and message name errors ([#654](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/654))
```
Comment (MaxTru) [inline CHANGELOG.md:11]:
```suggestion
* `FEAT`: add show callbacks to show error code and message name errors ([#654](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/654))
```

**Review by MaxTru — CHANGES_REQUESTED** [review]
What about

* `FIX`: add properties for `Condition` ([#2866](https://github.com/camunda/camunda-modeler/issues/2866))
* `FEAT`: show element template icons in header (https://github.com/bpmn-io/bpmn-js-properties-panel/commit/de8479f6ed08784503f2e4da5df3b1761c0cc1e3)
* `FEAT`: add icon property during element template application (https://github.com/bpmn-io/bpmn-js-properties-panel/commit/e48b6a21b28227c8ff18194f7111e81d01dce940)
* `CHORE`: all element template choose in dev setup (https://github.com/bpmn-io/bpmn-js-properties-panel/commit/e47642dc8623da965939318c69a0f06e4cca580b)

**Comment by philippfromme** [comment]
Ooops. 🤦🏻

**Comment by barmac** [comment]
How about if we wait for https://github.com/bpmn-io/bpmn-js-properties-panel/pull/656?

**Comment by MaxTru** [comment]
> How about if we wait for #656?

Makes sense, can you do the review @barmac ?

**Comment by barmac** [comment]
Yeah, I am on it right now.

**Comment by barmac** [comment]
The linked PR will take some time so let's skip it for this release.

**Review by MaxTru — APPROVED** [review]
lgtm

**Pull request merged**